chapter_15_advanced_forms.asciidoc

```diff
@@ -226,7 +231,7 @@ def adder(a, b):
 ----
 
 One way of putting it is that you should trust yourself not to do something
-'deliberately' stupid, but not something 'accidentally' stupid.
+'deliberately' irrational, but not something 'accidentally' silly.
```

Comment:
this sentence only really works if it's the same word each time, it's about the contrast between _deliberately_ and _accidentally_

```suggestion
_deliberately_ silly, but not something _accidentally_ silly.
```